packages/fiori-elements-writer/test/lrop.test.ts

```diff
@@ -208,6 +208,22 @@ describe(`Fiori Elements template: ${TEST_NAME}`, () => {
                 }),
                 service: v2Service
             } as FioriElementsApp<LROPSettings>
+        },
```

Comment:
The test is fine however for test simplification, this could be a separate test in which we do not dump the whole project into a snapshot but just check that the index file is not generated and the `package.json` does not contain the script

Reply:
No objection to this, to be honest I followed the test pattern here. Would it be ok to create a tbi for this? I need this PR merged if possible today.  I think we can do the same for other tests as well.

Reply:
Yes, that doesn't need to be part of this PR. We need a separate task to review our tests and make sure that they are still maintainable.